opentracing-util/src/test/java/io/opentracing/util/GlobalTracerTest.java

```diff
@@ -58,7 +48,9 @@ public void testGet_SingletonReference() {
 
     @Test
     public void testMultipleRegistrations() {
-        GlobalTracer.register(mock(Tracer.class));
+        assertThat(GlobalTracer.registerIfAbsent(provide(mock(Tracer.class))), is(true));
```

Comment:
@sjoerdtalsma to be sure could you please keep old tests for `register` method? And add new for new?